### PR TITLE
Fix get correct version from if match

### PR DESCRIPTION
**Pull request by fduch2k**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@fduch2k__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: test/utilities/strings.tests.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var expect = require('chai').expect,
+    strings = require('../../src/utilities/strings');
+
+describe('azure-mobile-apps.utilities.strings', function () {
+    describe('getVersiomFromIfMatch', function () {
```
Comment (danderson00) [inline test/utilities/strings.tests.js]: Nit picking: Name of test should be extractEtag. Happy to merge and resolve issue after this.

Comment (fduch2k) [inline test/utilities/strings.tests.js]: fixed

**Comment by danderson00** [comment]
Many thanks for the contribution!

Resolves https://github.com/Azure/azure-mobile-apps-node/issues/45

**Pull request merged**